sovtokenfees/sovtokenfees/test/req_handlers/conftest.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+import pytest
+from sovtoken.constants import UTXO_CACHE_LABEL
+from sovtoken.utxo_cache import UTXOCache
+from sovtokenfees.req_handlers.fees_utils import BatchController
+
+from indy_common.constants import CONFIG_LEDGER_ID
+
+from indy_node.test.request_handlers.helper import get_fake_ledger
+from sovtoken.test.req_handlers.conftest import db_manager as dbm
+
+from common.serializers import serialization
+from plenum.common.constants import KeyValueStorageType, BLS_LABEL
+from plenum.test.testing_utils import FakeSomething
+from state.pruning_state import PruningState
+from storage.helper import initKeyValueStorage
+
+
+@pytest.fixture(scope="module")
+def db_manager(tconf):
+    _db_manager = dbm(tconf)
+    storage = initKeyValueStorage(KeyValueStorageType.Memory,
+                                  None,
+                                  "configInMemoryStore",
+                                  txn_serializer=serialization.multi_sig_store_serializer)
```

Comment:
Why `multi_sig_store_serializer` is used here?

Reply:
replaced by in_memory_serializer

Reply:
serializer probably mentinoed here because of usage of inMemoryStore